### PR TITLE
zh-trans: update docs/concepts/cluster-administration/kubelet-garbage-collection.md

### PR DESCRIPTION
**Pull request by SataQiu**
zh-trans: update docs/concepts/cluster-administration/kubelet-garbage-collection.md

修复序号顺序被注释打断的问题

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md [hunk 1]
```diff
@@ -151,17 +151,17 @@ Users can adjust the following thresholds to tune image garbage collection with
 
 1. `image-gc-high-threshold`，触发镜像垃圾回收的磁盘使用率百分比。默认值为 85%。
 
-<!--
-1. `image-gc-high-threshold`, the percent of disk usage which triggers image garbage collection.
-Default is 85%.
--->
+    <!--
+    1.`image-gc-high-threshold`, the percent of disk usage which triggers image garbage collection.
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md]: looks like you need to add a space after `1.`.

Comment (SataQiu) [reply content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md]: I know what you mean, but if adding a space after `1.`, the display will be incorrect.
Is there a better way to fix it?
image

Comment (SataQiu) [reply content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md]: https://deploy-preview-11875--k8s-v1-12.netlify.com/zh/docs/concepts/cluster-administration/kubelet-garbage-collection/

Comment (tengqm) [reply content/zh/docs/concepts/cluster-administration/kubelet-garbage-collection.md]: hugo is not a perfect processor yet, so we have to live with it.
Please try move all the commented contents out of the numbered list.

**Comment by Mr-Linus** [comment]
errors
There is also an error in the English page, you can try to fix it. 😆 
/lgtm

**Comment by chenrui333** [comment]
这个 markdown 的序号syntax 确实是个有趣的问题。

/lgtm
/approve

**Pull request merged**